backend/user-service/app/types/userService.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+import { HTTP_STATUS } from "./http";
+import { userSchema } from "./user";
+
+// TODO: TBC
+export const getUserSchema = z.union([
```

Comment:
you don't need this file i think

Reply:
Resolved 437fbfc27a2f84c70a602983e4bd6c7a1753f4d6